<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GSAddSheldule</class>
 <widget class="QDialog" name="GSAddSheldule">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/sheldule</normalon>
   </iconset>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>411</width>
     <height>461</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="centralLayout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Псевдоним</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="aliasEdit"/>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabOnce">
       <attribute name="title">
        <string>Дата</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>391</width>
          <height>321</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="onceLayout">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QCalendarWidget" name="calendarOnce">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Дата и время</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="repeatCheckBox">
             <property name="maximumSize">
              <size>
               <width>100</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Повторять</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QDateTimeEdit" name="dateTimeOnce">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="repeatTimeComboBox">
             <item>
              <property name="text">
               <string>День</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Неделю</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Месяц</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Год</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="repeatLabel">
             <property name="text">
              <string>Повторять через:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tabWeekly">
       <attribute name="title">
        <string>Неделя</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>431</width>
          <height>281</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="weeklyLayout">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QTableWidget" name="weeklyTable">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="gridStyle">
            <enum>Qt::DotLine</enum>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <property name="rowCount">
            <number>7</number>
           </property>
           <property name="columnCount">
            <number>1</number>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>30</number>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderShowSortIndicator" stdset="0">
            <bool>false</bool>
           </attribute>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <row/>
           <column/>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Следующее время срабатывания:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="nextDateTimelabel">
        <property name="text">
         <string>никогда</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GSAddSheldule</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GSAddSheldule</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
